walletd/examples/wd_quickstart_guide.rs

```diff
@@ -2,8 +2,11 @@ use bdk::blockchain::ElectrumBlockchain;
 use bdk::electrum_client::Client;
 use bdk::keys::bip39::Mnemonic;
 use walletd_bitcoin::prelude::*;
+use walletd_ethereum::ethers::providers::Provider;
```

Comment:
Could be moved to prelude.rs

Reply:
these will get reworked over the next while. im expecting that we will have only the walletd prelude include in walletd examples